src/librustc_back/target/dragonfly_base.rs

```diff
@@ -16,20 +16,10 @@ pub fn opts() -> TargetOptions {
         linker: "cc".to_string(),
         dynamic_linking: true,
         executables: true,
-        linker_is_gnu: true,
         has_rpath: true,
-        pre_link_args: vec!(
-            "-L/usr/local/lib".to_string(),
-            "-L/usr/lib/gcc47".to_string(),
-            // GNU-style linkers will use this to omit linking to libraries
-            // which don't actually fulfill any relocations, but only for
-            // libraries which follow this flag.  Thus, use it before
-            // specifying libraries to link to.
-            "-Wl,--as-needed".to_string(),
```

Comment:
This is actually pretty useful for slimming down unneeded dynamic dependencies if it's an accepted flag, so just confirming, this isn't accepted on dragonfly?